src/app/products/loan-products/loan-product-stepper/loan-product-terms-step/loan-product-terms-step.component.html

```diff
@@ -87,7 +87,11 @@ <h4 fxFlex="98%" class="mat-h4">Number of repayments <i class="fas fa-question"
 
     <h3 fxFlex="23%" class="mat-h3">Interest Rates</h3>
 
-    <mat-checkbox fxFlex="73%" labelPosition="before" formControlName="isLinkedToFloatingInterestRates">
+    <mat-checkbox [formControl]="zeroInterest" fxFlex="23%" labelPosition="before" *ngIf="!loanProductTermsForm.value.isLinkedToFloatingInterestRates">
```

Comment:
Please raise a separate PR for this "zero interest" enhancement!

Reply:
The zero interest is part of the PR I mean it is required for identifying If the Loan product is zero interest and If this has Adv Pment Alloc we can display the Fixed Length input